### ISSUE TITLE
Docker compose has some wrong configurations

### ISSUE THREAD

**Issue by marcosvlehmkuhl** [TallerWebSolutions/drupal-enterprise-boilerplate#3]
Some volume configurations on docker compose are mounting volumes to wrong destination.

E.g:
```
volumes:
      - $HOME/.ssh:/drupal/.ssh
      - $HOME/.gitconfig:/drupal/.gitconfig
```

**Comment by marcosvlehmkuhl** [comment]
https://github.com/TallerWebSolutions/drupal-enterprise-boilerplate/pull/4

**Comment by lhguerra** [comment]
Fixed by PR #4
